### PR TITLE
chore: Port missing tokens to style-dictionary

### PR DESCRIPTION
**Pull request by afontcu**
This PR doesn't add any feature, it simply ports missing tokens (shadows, breakpoints…) to style-dictionary and cleans up its folder structure a bit.

We also changed the output SCSS to make it easier to eventually add CSS custom properties (incoming PR).

Notice I also moved style-dictionary stuff into `.style-dictionary/` to mimic storybook and other "helper" utilities that are not "source files" of the UI library.

Also notice the PR is not a breaking change because get-color and get-space did not change their behavior, and that we're still exposing shadow vars:

```scss
$shadow-light: rgba(0, 0, 0, 0.027451) !default;
$shadow-medium: rgba(0, 0, 0, 0.0980392) !default;
$shadow-dark: rgba(0, 0, 0, 0.498) !default;
```

### CODE REVIEW

File: .style-dictionary/transformToStyleDictionary.spec.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import transformToStyleDictionary from './transformToStyleDictionary'
```
Comment (Samitier) [inline .style-dictionary/transformToStyleDictionary.spec.js:1]: if `.style-dictionary` dir is autogenerated, shouldn't we have this test somewhere else?

Comment (Samitier) [reply .style-dictionary/transformToStyleDictionary.spec.js:1]: nevermind, I didn't understand the PR 😂

**Comment by Samitier** [comment]
maybe it's a good moment to re think if this shadow variables are really useful or not. Why don't they have the rest of the box-shadow props (not only the color)?

**Review by Samitier — APPROVED** [review]

**Review by alexcastelo95 — APPROVED** [review]

**Comment by afontcu** [comment]
> maybe it's a good moment to re think if this shadow variables are really useful or not

I wanted to keep this PR non-breaking so we could merge it without any consequences. I'm not 100% sure if shadow variables are being used and where (I assume they are), so we might need to investigate a bit before removing them 👍 

That being said, I don't care that much about current shadows 😂 they are not being used consistently and do not convey the expected meaning ("layers" of depth), so… we really need to talk with design folks about getting them right (#305)

**Pull request merged**